docs/ansible/roles/fhs/man_seealso.rst

```diff
@@ -5,7 +5,7 @@
 See Also
 ========
 
-- Manual pages: :man:`hier(7)`
+- Manual pages: :man:`fhs(7)`
```

Comment:
There is no `fhs(7)` manpage. The `hier(7)` is correct here.

Reply:
And I learned something new :) Now I know how you picked the name. So the name "hier" has some background. I removed this commit from the PR. It is not needed in the history.